README.md

```diff
@@ -25,3 +25,6 @@ yarn test:dev
 
 One technique that can helpful when writing tests, is to reference the coverage report for the file/function/feature you're testing. To do this, run `yarn test:dev` from your terminal. This will generate a coverage report for the project. The `coverage` directory will be located in the root directory. There is an `index.html` which shows the coverage report per file.
 
+## Contributing
+
+Please see the [Contributing Guidlines](CONTRIBUTING.md) document for this repo's specific contributing guidelines.
```

Comment:
If would be nice to link out to the gitbook site as well for general contributing guidelines.

Reply:
I'm not familiar with gitbook. I will leave that to someone who has more knowledge on it and can document it properly.